interoperability-layer-openhim/swarm.sh

```diff
@@ -60,112 +75,119 @@ verify_mongos() {
   done
 }
 
-prepare_console_config() {
+function prepare_console_config() {
   # Set host in OpenHIM console config
   sed -i "s/localhost/${OPENHIM_CORE_MEDIATOR_HOSTNAME}/g; s/8080/${OPENHIM_MEDIATOR_API_PORT}/g" /instant/interoperability-layer-openhim/importer/volume/default.json
 }
 
-main() {
-  if [[ "${STATEFUL_NODES}" == "cluster" ]]; then
-    log info "Running Interoperability Layer OpenHIM package in Cluster node mode"
-    mongo_cluster_compose_param="-c ${COMPOSE_FILE_PATH}/docker-compose-mongo.cluster.yml"
-  else
-    log info "Running Interoperability Layer OpenHIM package in Single node mode"
-    mongo_cluster_compose_param=""
-  fi
-
-  if [[ "${MODE}" == "dev" ]]; then
-    log info "Running Interoperability Layer OpenHIM package in DEV mode"
-    local mongo_dev_compose_param="-c ${COMPOSE_FILE_PATH}/docker-compose-mongo.dev.yml"
-    local openhim_dev_compose_param="-c ${COMPOSE_FILE_PATH}/docker-compose.dev.yml"
-  else
-    log info "Running Interoperability Layer OpenHIM package in PROD mode"
-    local mongo_dev_compose_param=""
-    local openhim_dev_compose_param=""
-  fi
-
-  if [[ "${ACTION}" == "init" ]]; then
-    config::set_config_digests "${COMPOSE_FILE_PATH}"/docker-compose.yml
-    config::set_config_digests "${COMPOSE_FILE_PATH}"/importer/docker-compose.config.yml
-
-    try "docker stack deploy -c ${COMPOSE_FILE_PATH}/docker-compose-mongo.yml $mongo_cluster_compose_param $mongo_dev_compose_param instant" "Failed to deploy mongo"
+function init_package() {
```

Comment:
Please be consistent with other packages, choose one, either `init_package` or `initialize_package`

Reply:
cool done